resources/idlharness.js

```diff
@@ -840,9 +840,9 @@ IdlInterface.prototype.test_self = function()
         var desc = Object.getOwnPropertyDescriptor(self, this.name);
         assert_false("get" in desc, "self's property " + format_value(this.name) + " has getter");
         assert_false("set" in desc, "self's property " + format_value(this.name) + " has setter");
-        assert_true(desc.writable, "self's property " + format_value(this.name) + " is not writable");
+        assert_true(desc.writable, "self's property " + format_value(this.name) + " is writable");
         assert_false(desc.enumerable, "self's property " + format_value(this.name) + " is enumerable");
-        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is not configurable");
+        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is configurable");
```

Comment:
They look OK to me. The error message is going to say something like

    self's property foo is enumerable: expected false got true

which to me reads as "desc.enumerable should be false, but it is true". Are you suggesting negating that sentece instead?